### DISCUSSION TITLE
How to control another EV3 device from a master EV3 in pybricks?

### DISCUSSION THREAD

**Discussion by pekyeen** [pybricks/support#1063] [Q&A]
I am new to EV3, please bear with me. I successful connect a single EV3 device and program it with Visual Studio Code (VSC). When I do a daisy chain between EV3-A and EV3-B, I connect EV3-A to VSC, may I know how do control the EV3-B's motor port from the micropython?  I read from Scratch project, the second layer port can be setup as variable with this formula layer x 100 + port. Port A = 1, Port B = 2 and so on. But in Pybricks, the default port is A to D for EV3-A (Motor(Port.A)). How about the motor ports from EV3-B? Is it possible to configure? Appreciate if some one do have answers for this. Thank You..

**Comment by laurensvalk** [comment]
USB daisy chaining as in the original LEGO app is not supported in Pybricks, but you can use [Bluetooth messaging](https://pybricks.com/ev3-micropython/messaging.html) to send messages between the robots.

**Reply by pekyeen** [reply]
Thanks for the reply. If EV3-A is connected to WiFi, can still Bluetooth between EV3-A and EV3-B be done? Both WiFi and Bluetooth should be independent, right?

**Reply by laurensvalk** [reply]
Yes, combining WiFi/Bluetooth should be fine.

I think it should even work if one hub is already connected to Visual Studio Code with Bluetooth, but I have not tried this recently.

**Reply by pekyeen** [reply]
Thanks again. Let me try it out. :)
